### PR TITLE
Extend the number of waypoints

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: latest.py [hunk 1]
```diff
@@ -8,22 +7,25 @@ def main():
     fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
     goto( -920,  920, 0)
     goto( -920,-1450, 0)
-    pick_up()
-    goto(  750,-1200, 0)
-    set_down()
-    goto( 1000,-1300, 1)
```
Comment (kreier) [inline latest.py:14]: Some `pick_up()` and `set_down()` seem to be removed - to save time?

File: latest.py [hunk 2]
```diff
@@ -8,22 +7,25 @@ def main():
...
+    goto(  750,-1400, 0) # blue in right zone
+    goto(  500,-1150, 1)
+#   goto(    0, -950, 0) # m = - 2/5
```
Comment (kreier) [inline latest.py:12]: Why is the `# goto( 0. -950, 0)` still here?

File: latest.py [hunk 3]
```diff
@@ -8,22 +7,25 @@ def main():
...
+    goto(    0,    0, 0) # m = - 5/8
+    goto( -600,  200, 0) # yellow in left zone
+    goto(  400,  400, 1)
+    goto(    0,  950, 0)
+    goto( -600,  950, 0) # yellow in left zone
+    goto(  920,  950, 1)
     goto(  920, 1500, 0)
-    pick_up()
-    goto( -600, 1320, 0)
-    set_down()
```
Comment (kreier) [inline latest.py:18]: One more `pick_up()` and `set_down()` pair removed.

File: latest.py [hunk 4]
```diff
@@ -50,10 +52,8 @@ def goto(target_x, target_y, reverse):
         drivetrain.drive_for(REVERSE, distance, MM, wait=True)
     else:
         drivetrain.drive_for(FORWARD, distance, MM, wait=True)
-
 def pick_up():
     fork_motor_group.spin_to_position(1500, DEGREES, wait=True)
-
 def set_down():
     fork_motor_group.spin_to_position(1800, DEGREES, wait=True)
```
Comment (kreier) [inline latest.py:58]: 60 more points (from 70 to 130) with the same 60 lines by removing empty lines?

File: latest.py (hunk 1 above)
Comment (ssis-robotics) [reply latest.py:14]: yes. It's about 0.7 seconds for `pick_up()` and `set_down()` and moving forward you track the goal just with you

File: latest.py (hunk 3 above)
Comment (ssis-robotics) [reply latest.py:18]: same as above (line 11 to 14)

File: latest.py (hunk 4 above)
Comment (ssis-robotics) [reply latest.py:58]: That's the goal

File: latest.py (hunk 3 above)
Comment (ssis-robotics) [reply latest.py:18]: same as above (line 11 to 14)

File: latest.py (hunk 4 above)
Comment (ssis-robotics) [reply latest.py:58]: That's the goal

**Comment by ssis-robotics** [comment]
With comments - this could be merged!

**Comment by ssis-robotics** [comment]
With comments - this could be merged!

**Pull request merged**